php/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer as builder
+WORKDIR /tmp/composer-deps
+COPY composer.* ./
+RUN composer install
+
+FROM php:8.2-cli-alpine3.17 as runner
+COPY --from=builder /tmp/composer-deps /app/vendor
+COPY . /app/
```

Comment:
Building things like the .env file into the docker image isn't great, it would be better to load the env file using docker compose when running in docker (otherwise the image has to be re-built to change settings)